lib/Runtime/Executor/ThreadPoolExecutor.h

```diff
@@ -127,8 +125,7 @@ class ExecutionState final {
       inputCtxs_;
   /// Placeholders for tensors generated by DAG nodes that aren't the final
   /// output (i.e. they have children). The set of currently executing nodes.
```

Comment:
Add a comment about lack of ownership of tensors here

Reply:
What do you mean here? The Placeholders are owned by `module_`. (maybe just write the comment you like here and i'll copy it in 😄)